packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts

```diff
@@ -1,8 +1,25 @@
 import { css } from '@emotion/react';
-import { neutral, success, textSans } from '@guardian/source-foundations';
-import type { LabelPosition } from './ToggleSwitch';
+import type { SerializedStyles } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	neutral,
+	space,
+	success,
+	textSans,
+} from '@guardian/source-foundations';
+import type {
+	LabelPosition,
+	ToggleSwitchFontSize,
+	ToggleSwitchFontWeight,
+} from './ToggleSwitch';
 
-export const buttonStyles = (labelPosition: LabelPosition) => css`
+const toggleBackground = 'rgba(255, 255, 255, 0.4)';
+const toggleBorder = 'rgba(255, 255, 255, 0.6)';
+const toggleBorderGreen = '#A7CFB8';
```

Comment:
is this something that our palette doesn't provide?

Reply:
yes.

Reply:
okay, maybe something to double check with Akemi whether it belongs in palette? (sorry if you already have)

Reply:
(its the colour we would get by layering 40% opacity white border over success[400], but because borders don't sit over the thing they surround, we have to pre-calculate that colour and apply as a standard border)

Reply:
thanks for the explanation, that makes sense to me :) 

maybe a small comment above it with that layering method would be helpful, so we remember why it's not coming from the palette